code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -338,6 +341,14 @@
 
 	load(AM)
 
+	if(user.incapacitated() || !(L.mobility_flags & MOBILITY_STAND))
+		return
+
+	if(!istype(AM))
+		return
+
+	load(AM)
+
```

Comment:
This entire section is duplicated after the first load(AM)

Reply:
the two checks are different, look again

Reply:
Well you already loaded the AM before these checks. What purpose do they serve?

Reply:
i have no idea. i just know the two checks are different, they were there before i fiddled with anything

Reply:
Your diff says otherwise: https://github.com/tgstation/tgstation/pull/46078/files
Green lines are added by this PR, notably after the first load(AM)

Reply:
huh.

Reply:
huh